### PR TITLE
remove mapfishapp from config

### PR DESCRIPTION
**Pull request by fvanderbiest**
Proposal for  #252

### CODE REVIEW

File: datahub/conf/default.toml [hunk 1]
```diff
@@ -77,8 +77,6 @@ max_zoom = 14
 # ${layer_name}: Name of the layer
 # Be careful to use englobing single quotes, if your template syntax includes JSON (with double quotes)
 # Examples:
-# mapfishapp template:
-# external_viewer_url_template = 'https://dev.geo2france.fr/mapfishapp/?owsurl=${service_url}&layername=${layer_name}&owstype=${service_type}Layer'
```
Comment (landryb) [inline datahub/conf/default.toml:81]: same thing ?

Comment (fvanderbiest) [reply datahub/conf/default.toml:81]: hmm, just below ? ;-)

File: README.md [hunk 2]
```diff
@@ -54,6 +54,5 @@ There are plenty of other configuration options available, so feel free to brows
 
 We do recommend that you:
  * change your SDI logo, with [header/logo.png](header/logo.png)
- * update the viewer config with [mapfishapp/js/GEOR_custom.js](mapfishapp/js/GEOR_custom.js)
```
[the remark this answers is not in the document]
Comment (fvanderbiest) [reply README.md:57]: good point

File: datahub/conf/default.toml (hunk 1 above)
Comment (landryb) [reply datahub/conf/default.toml:81]: doh! sorry :)

**Pull request merged**